automation/video.py

```
@@ -207,37 +252,72 @@ async def play_video(self, video_url: str, video_selector: str = "video",
+                                description=f"[cyan]{self.format_time(current_time)}[/cyan]/[dim]{self.format_time(video_duration)}[/dim]"
+                            )
+                    else:
+                        # 无法获取视频状态时
+                        progress.update(task, description=f"[yellow]等待中 {self.format_time(elapsed)}[/yellow]")
+
+                    # 尝试自动延长会话
+                    await self.auth_manager.refresh_cookies()
+
+                    # 检查Cookie是否有效
+                    if not await self.auth_manager.check_cookie_validity():
+                        console.print("[red]⚠ Cookie已失效，停止观看视频[/red]")
```

Ruff (0.14.10)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)